\documentclass{article}

\setcounter{page}{1}
\setlength{\oddsidemargin}{0.in}
\setlength{\evensidemargin}{0.in}
\setlength{\textwidth}{6.5in}
\setlength{\topmargin}{-0.5in}
\setlength{\headheight}{0.17in}
\setlength{\headsep}{0.35in}
\setlength{\textheight}{9.in}
\setlength{\parindent}{0in}
\setlength{\parskip}{.1in}
\usepackage[T1]{fontenc}
\usepackage[round]{natbib}
\bibliographystyle{../../bst/sysbio}
\usepackage[fleqn]{amsmath}
\usepackage{amssymb}
\usepackage{xspace}
\usepackage{verbatim}
\usepackage{paralist}
\usepackage{setspace}
\usepackage{graphicx}
\usepackage{caption}
\usepackage{subcaption}
\usepackage{float}
\usepackage{url}
\usepackage{hyperref}
\hypersetup{backref, urlcolor=blue, linkcolor=blue, citecolor=blue,  colorlinks=true}

% use the fancyhdr package to allow both headers and footers
\usepackage{fancyhdr}
\pagestyle{fancy}
\fancyhf{} % clear all headers and footers
\fancyhead[RO,LE]{BEAST v1.7.5 Divergence-time Estimation Demo}
% \fancyhead[RO,LE]{2012}
\fancyfoot[LO,LE]{\footnotesize This work is licensed under a \href{http://creativecommons.org/licenses/by/4.0/deed.en_US}{Creative Commons Attribution 4.0 International License.}}
\fancyfoot[CE,CO]{}
\fancyfoot[RO,RE]{\thepage}
\renewcommand{\footrulewidth}{0.5pt}

% \floatstyle{boxed}
% \restylefloat{figure}
% \restylefloat{table}
\newfloat{textbox}{hptb}{lotb}
\floatname{textbox}{Box}

\newcommand{\execmd}[1]{\texttt{#1}\\}
\newcommand{\cmdopt}[1]{\texttt{#1}\xspace}
\newcommand{\cmd}[1]{\texttt{#1}\xspace}
\newcommand{\localfile}[1]{\textsf{#1}\xspace}
\newcommand{\QandA}[2]{\textit{#1}\footnote{#2}\xspace}
\newcommand{\todo}[1]{{\color{red}TODO: {#1}}\xspace}
\newcommand{\menutab}[1]{\textbf{\textit{#1}}\xspace}
\newcommand{\subItem}[2]{\menutab{#1}$\rightarrow$\menutab{#2}\xspace}
\newcommand{\subsubItem}[3]{\menutab{#1}$\rightarrow$\menutab{#2}$\rightarrow$\menutab{#3}\xspace}
\newcommand{\program}[1]{#1\xspace}
\newcommand{\plusbutton}{``$\boldsymbol{+}$''\xspace}
\newcommand{\beast}{\href{http://beast.bio.ed.ac.uk/Main_Page}{\program{BEAST}}\xspace}
\newcommand{\taxonset}[1]{{\bfseries\sffamily #1}\xspace}
\newcommand{\field}[1]{\texttt{#1}\xspace}
\newcommand{\fieldvalue}[1]{{\bfseries\sffamily #1}\xspace}
\newcommand{\questionfont}[1]{{\bfseries\sffamily\textit{#1}}\xspace}
\newcommand{\question}[1]{{\questionfont{#1}\\\vspace{2cm}}\xspace}


\begin{document}
{\Large\bf BEAST Divergence-time Lab}
{\singlespacing \tableofcontents}
\newpage
%###########################################################################################

\section{Objective}
The goal of this tutorial is to help you become familiar with using fossil data
to estimate time-calibrated phylogenetic trees in a Bayesian framework.
We will use an example dataset of DNA sequences of crocodylians (crocodiles and alligators)
and the software package \beast version 1.7.5.
We will work through the steps necessary for estimating the phylogenetic
relationships among crocodiles and alligators, while simultaneously using
fossil information and relaxed-clock models to calibrate the branches on the
tree to units of time (millions of years).

The stylistic conventions for this tutorial:

\begin{tabular}{|p{3.2in}|p{2.8in}|}
\hline	Names of files & \localfile{in this font}.\\
\hline	Web site URLs and other clickable links &  \href{http://www.google.com}{look like this}.\\
\hline	Nested menu items & \subsubItem{Top Name}{Mid Name}{Lowest}\\
\hline	\program{BEAUTi} menu tabs & \menutab{tab name}\\
\hline	\program{BEAUTi} option fields & \field{field name}\\
\hline	Field values that you enter & \fieldvalue{field value}\\
\hline	Questions for you to answer & \questionfont{look like this?}\\
\hline
\end{tabular}

\section{Version, Author information, and Acknowledgements}
This tutorial was written by Jamie Oaks and Melissa Callahan for \beast version
1.7.5.
Many of the ideas for this lab came from a great
\href{http://treethinkers.org/divergence-time-estimation-using-beast/}{tutorial}
by \href{http://phylo.bio.ku.edu/content/tracy-heath}{Tracy Heath}.

\section{Background Information}
Often biologists are interested in estimating trees in which the branch lengths
are proportional to time.
However, because we only observe the product of time and the rate of evolution
(differences among species), time and rate are inextricably linked.
As a result, we have to make some assumptions about how the rate of evolution
has changed over evolutionary history in order to tease apart the contributions
of rate and time, and estimate branch lengths proportional to time.
The simplest assumption (i.e., model) is that the rate of substitution is constant
over time; a global strict clock \citep{Zuckerkandl1962}.
However, we know that in most situations this assumption is violated.
As a result, there has been a lot of work done on developing ``relaxed-clock''
models that allow the rate of substitution to change over time and among
lineages.
These methods allow us to estimate relative divergence times across a phylogeny
(Figure~\ref{fig:branchLengths}).

    \begin{figure}[htbp]
        \centering
        \fbox{\includegraphics[width=0.8\textwidth]{../../images/branch-lengths.jpg}}
        \caption{The branch lengths of a phylogenetic tree are the product of rate and time.}
        \label{fig:branchLengths}
    \end{figure}

In order to estimate absolute divergence times, we need extra information to
calibrate the rate of evolution across the tree.
Paleontological data (e.g., fossils) are often used for this purpose.
If we know the date of a fossil, and can estimate its relationships to the
extant species, we can use this information to constrain the age of a node
in the tree.
Combining this approach with a relaxed-clock model allows us to estimate
a phylogeny in which the branch lengths are in units of time.

\section{Programs Used in This Lab}
We will be using the free, open-source software package, \beast (Bayesian
Evolutionary Analysis Sampling Trees;
\href{http://beast.bio.ed.ac.uk}{\url{http://beast.bio.ed.ac.uk}}), for
estimating divergence times under a relaxed clock.
\beast offers a wide range of models for analyzing molecular data in
a Bayesian statistical framework.
\beast also comes with several other utility programs that we will be also
using to prepare input files (\program{BEAUTi}; Bayesian Evolutionary Analysis
Utility) and summarize output files (\program{LogCombiner and TreeAnnotator}).
We will also be using the programs \program{Tracer}
(\href{http://tree.bio.ed.ac.uk/software/tracer}{\url{http://tree.bio.ed.ac.uk/software/tracer}})
and \program{FigTree}
(\href{http://tree.bio.ed.ac.uk/software/figtree}{\url{http://tree.bio.ed.ac.uk/software/figtree}})
for evaluating, summarizing, and viewing results.

\section{The Data}
We will be analyzing DNA sequence data from 24 species of crocodylians
(crocodiles, alligators, caimans, and gharials).
The sequences are of the protein-coding \emph{cytochrome b} gene from the
mitochondrial genome.
For these data, a likelihood ratio test rejects a global strict clock model.
So, we will use an uncorrelated log-normally distributed (UCLD) relaxed-clock
model \citep{Drummond2006} to account for variation in rates of substitution
across the tree in order to estimate branches proportional to time.

Crocodylians have a rich fossil record, and we will be using some of this
fossil information to specify priors on the ages of certain nodes in the
phylogeny (Figure~\ref{fig:crocFossils}).
This will calibrate the branch-specific rates of the relaxed-clock model, and
scale the branches of the tree to absolute time.
We will use units of millions of years for specifying the node-age
calibrations, and so divergence-time estimates will also be in units of
millions of years.

    \begin{figure}[htbp]
        \centering
        \fbox{\includegraphics[width=0.8\textwidth]{../../images/crocodylia-fossils.pdf}}
        \caption{A phylogenetic estimate of Crocodylia with rough fossil placements.}
        \label{fig:crocFossils}
    \end{figure}

\section{Tutorial}
\newcounter{stepCounter}
\newcommand{\step}[2]{\addtocounter{stepCounter}{1} {\bf \hypertarget{step\arabic{stepCounter}}{Step \arabic{stepCounter}:}}\xspace #2\par}
\newcommand{\intermediate}[1]{#1}
\input{div-time-tutorial-steps.tex}

\newpage
\section{Quick Version of the Tutorial}
\setcounter{stepCounter}{0}
\renewcommand{\step}[2]{\addtocounter{stepCounter}{1} {\bf \hyperlink{step\arabic{stepCounter}}{Step \arabic{stepCounter}}:}\xspace #1\par}
\renewcommand{\intermediate}[1]{}
\input{div-time-tutorial-steps.tex}

\newpage
\section{Additional Information/Resources}
Great resources for divergence-time estimation include a
\href{http://treethinkers.org/divergence-time-estimation-using-beast/}{tutorial}
and
\href{https://molevol.mbl.edu/wiki/images/6/6f/Bodega_2013_divtime_lecture.pdf}{slides}
by \href{http://phylo.bio.ku.edu/content/tracy-heath}{Tracy Heath}!

\newpage
\bibliography{../../bib/references}

\end{document}

